extensions/wikia/FounderEmails/events/FounderEmailsCompleteDigestEvent.class.php

```diff
@@ -27,10 +27,15 @@ public function process ( Array $events ) {
 		$wgTitle = Title::newMainPage();
 		$founderEmailObj = FounderEmails::getInstance();
 		// Get list of founders with digest mode turned on (defined in FounderEmailsEvent)
-		$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-complete-digest' );
+		if ( empty( $events ) ) {
+			$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-complete-digest' );
+		} else {
+			$cityList = $events;
+		}
 		$wikiService = new WikiService();
 
 		foreach ( $cityList as $cityID ) {
+			Wikia::initAsyncRequest( $cityID );
```

Comment:
This fix isn't about explicit requests for a wiki ID its for code that implicitly assumes it is running in the context of a particular wiki. For example, the two lines you identified won't help wfMessage correctly translate ```{{SITENAME}}``` when it appears in a string (the error happening in this ticket).  For things like this we need to set global variables.